kotskinds/apis/kots/v1beta1/installation_types.go

```diff
@@ -61,6 +61,23 @@ type Installation struct {
 	Status InstallationStatus `json:"status,omitempty"`
 }
 
+// Merge known images by overwriting existing image entries with newer ones.
+func (i *Installation) MergeKnownImages(newImages []InstallationImage) {
```

Comment:
instead of this function, wouldn't it be simpler to add a function [here](https://github.com/replicatedhq/kots/blob/754fcd7d5c759713a1a2ada99920eb4ba657f193/pkg/upstream/installation.go) that reads the installation file before writing kots kinds?